src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -498,7 +498,6 @@
     Sfjs.addEventListener(window, 'load', function() {
         Sfjs.createTabs();
         Sfjs.createToggles();
-        Sfjs.renderAjaxRequests();
```

Comment:
This file is included in toolbar_js first.. so for being super fast this actually wins from the XHR response :))